source/class/cv/IconConfig.js

```diff
@@ -31,1023 +29,12350 @@ qx.Class.define('cv.IconConfig', {
+        }
+      },
+
+      control_3dot_vert_s: {
+        '*': {
+          white: '*/white',
+          ws: '*/white',
+          antimony: '*/blue',
+          boron: '*/green',
+          lithium: '*/red',
+          potassium: '*/purple',
+          sodium: '*/orange',
+          '*': { '*': cv.util.IconTools.svgKUF('control_3dot_vert_s') }
+        }
+      },
+
+      control_4: {
+        '*': {
+          white: '*/white',
+          ws: '*/white',
+          antimony: '*/blue',
+          boron: '*/green',
+          lithium: '*/red',
+          potassium: '*/purple',
+          sodium: '*/orange',
+          '*': { '*': cv.util.IconTools.svgKUF('control_4') }
+        }
+      },
+
+      control_all_on_off: {
+        '*': {
+          white: '*/white',
+          ws: '*/white',
+          antimony: '*/blue',
+          boron: '*/green',
+          lithium: '*/red',
+          potassium: '*/purple',
+          sodium: '*/orange',
+          '*': { '*': cv.util.IconTools.svgKUF('control_all_on_off') }
+        }
+      },
+
+      control_arrow_down_left: {
+        '*': {
+          white: '*/white',
+          ws: '*/white',
+          antimony: '*/blue',
+          boron: '*/green',
+          lithium: '*/red',
+          potassium: '*/purple',
+          sodium: '*/orange',
+          '*': { '*': cv.util.IconTools.svgKUF('control_arrow_down_left') }
+        }
+      },
+
+      control_arrow_down_right: {
+        '*': {
+          white: '*/white',
+          ws: '*/white',
+          antimony: '*/blue',
+          boron: '*/green',
+          lithium: '*/red',
+          potassium: '*/purple',
+          sodium: '*/orange',
+          '*': { '*': cv.util.IconTools.svgKUF('control_arrow_down_right') }
+        }
+      },
+
+      control_arrow_down: {
+        '*': {
+          white: '*/white',
+          ws: '*/white',
+          antimony: '*/blue',
+          boron: '*/green',
+          lithium: '*/red',
+          potassium: '*/purple',
+          sodium: '*/orange',
+          '*': { '*': cv.util.IconTools.svgKUF('control_arrow_down') }
+        }
+      },
+
+      control_arrow_downward: {
+        '*': {
+          white: '*/white',
+          ws: '*/white',
+          antimony: '*/blue',
+          boron: '*/green',
+          lithium: '*/red',
+          potassium: '*/purple',
+          sodium: '*/orange',
+          '*': { '*': cv.util.IconTools.svgKUF('control_arrow_downward') }
+        }
+      },
+
+      control_arrow_left: {
+        '*': {
+          white: '*/white',
+          ws: '*/white',
+          antimony: '*/blue',
+          boron: '*/green',
+          lithium: '*/red',
+          potassium: '*/purple',
+          sodium: '*/orange',
+          '*': { '*': cv.util.IconTools.svgKUF('control_arrow_left') }
+        }
+      },
+
+      control_arrow_leftward: {
+        '*': {
+          white: '*/white',
+          ws: '*/white',
+          antimony: '*/blue',
+    
```

Comment:
Probably we should treat this as a special case manually keep or upgrade the old file.